types/event-source-polyfill/index.d.ts

```diff
@@ -0,0 +1,82 @@
+// Type definitions for event-source-polyfill 1.0
+// Project: https://github.com/Yaffle/EventSource
+// Definitions by: Emily Klassen <https://github.com/forivall>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.4
```

Comment:
```suggestion
```
you can skip this, 3.7 is default now here

Reply:
locally, i'm getting this:
```
» yarn test event-source-polyfill
yarn run v1.22.17
$ dtslint types event-source-polyfill
Error: /Users/forivall/code/repos/DefinitelyTyped/types/event-source-polyfill/event-source-polyfill-tests.ts:20:7
ERROR: 20:7  expect  Compile error in typescript@3.3 but not in typescript@3.4.
Fix with a comment '// Minimum TypeScript Version: 3.4' just under the header.
Type 'EventSourcePolyfill' is not assignable to type 'EventSource'.
  Types of property 'onerror' are incompatible.
    Type '((this: EventSource, ev: Event) => any) | null' is not assignable to type '(evt: MessageEvent) => any'.
      Type 'null' is not assignable to type '(evt: MessageEvent) => any'.

    at /Users/forivall/code/repos/DefinitelyTyped/node_modules/dtslint/bin/index.js:206:19
    at Generator.next (<anonymous>)
    at fulfilled (/Users/forivall/code/repos/DefinitelyTyped/node_modules/dtslint/bin/index.js:6:58)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

i'll remove it and see if the failure happens in CI